ansible_ai_connect/main/middleware.py

```diff
@@ -165,8 +81,8 @@ def __call__(self, request):
                                 for task in tasks
                             ],
                             rh_user_org_id=getattr(request.user, "org_id", None),
-                            suggestion_id=request_suggestion_id,
-                            model_name=model_name,
+                            suggestion_id=getattr(response, "suggestionId", ""),
```

Comment:
There are scenarios where the suggestionId is in the request but doesn't make it into the response, such as if the request fails due to an authentication failure. This change would break that.